### ISSUE TITLE
Skipped Heading Level

### ISSUE THREAD

**Issue by spencercheng** [BYU-ODH/wp-humanities-byu#463]
To better provide structure and accessibility, headings should be in order. (h1, h2,h3, etc..)

- [x] Footer right after the content (Info and Additional Links) should be h2
- [x] Projects page cards title should be h2 (project cards template)
- [x] Testing labs cards title should be h2
- [x] Room's page - The card's title should be h2
- [x] Support's page - Title should be h2
- [x] Blogs page title should be h2 
- [x] Computer Classroom page title should be h2 (Links Under Rooms)
- [x] Foreign Language Activity Commons should be h2 (Links Under Rooms)
- [x] Small Groups Rooms and Classroom should be h2 (Links Under Rooms)
